catalog/core/catalog-core-impl/pubsub/src/test/java/ddf/catalog/pubsub/PubSubOgcFilterTest.java

```diff
@@ -89,6 +89,8 @@ public void testGeospatialFeatureEvaluate() throws TransformerException {
     assertFalse(bboxFilter1.evaluate(feature));
   }
 
+  @Test
+  @Ignore
```

Comment:
I think we should either delete this function or create a ticket for the TODO at the bottom of the function.

Reply:
👍 Removed. These tests have been ignored since the initial commit.